org.equeim.Tremotesf.json

```diff
@@ -0,0 +1,34 @@
+{
+    "app-id": "org.equeim.Tremotesf",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "tremotesf",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host:ro",
```

Comment:
Why RO access to entire host?

Reply:
Tremotesf can be used for local transmission-daemon instances or with mounted remote directories. In this case it can open torrent's files, show them in file manager (on flatpak using org.freedesktop.FileManager1 D-Bus interface) or select download directory using file chooser. For this it needs (readonly) access to filesystem (home directory only is not enough, since remote dirs can be mounted outside of it). I don't know is it is possible without filesystem access using portals, but currently Tremotesf checks for files existence before showing "Open" menu items, so it won't work without it.

Reply:
Thanks for explaination.